UserDocs.md

```diff
@@ -17,242 +17,228 @@ Install T4:
 
-```python
-df, meta = he.get("bucket-name/my-frame.parquet")
-```
-
-`get` returns a tuple of values. The first entry is the data that you put in. The second entry is the metadata associated with the object. If no metadata exists, this value will be `None`.
+    import t4
+	t4.list_packages()  # to see local packages
+	t4.list_packages("s3://bucket")  # to see remote packages
 
-To learn more about metadata, skip forward to the section on [Metadata and querying](#metadata-and-querying).
+<!-- ![](img/helium-api.png) -->
 
-### Working with files
 
-The commands for moving files and from T4 are very similar to the ones for moving objects in memory.
+### Managing individual files
 
-To put a file to a T4 object, use the `put_file` command:
+T4 provides a lower-level API for managing both local and remote files using `t4.copy`:
 
 ```python
-# Generate example data
-import pandas as pd
-import numpy as np
-df = pd.DataFrame(np.random.random((1000, 10)))
-df.to_csv("my-frame.csv")
-
-# Put it
-he.put_file("my-frame.csv", "bucket-name/my-frame.csv")
+t4.copy("file:///my-frame.csv", "s3://bucket-name/my-frame.csv")  # copy local to local
+t4.copy("s3:///my-frame.csv", "s3://bucket-name/my-frame.csv")  # copy remote to remote
+t4.copy("s3://my-frame.csv", "file:///bucket-name/my-frame.csv")  # copy remote to local
 ```
 
-This will populate a CSV file in T4. T4 supports files in any format, and up to 5 TB in size.
+T4 supports files in any format, and up to 5 TB in size.
 
-Similarly, to get a T4 object back, use the `get_file` command:
+To delete a remote file, use the `delete` command:
 
 ```python
-df, meta = he.get_file("bucket-name/my-frame.csv", "my-frame.csv")
+he.delete("s3://bucket-name/my-frame.csv")
 ```
 
-This will download a `my-frame.csv` to your local disk.
 
-Just like `he.get` , this will result in a tuple whose first element is the data and the second, the metadata. If no metadata is present, `None` will be returned.
-
-
-### Deleting files
+### Working with memory
 
-To delete a file that's already been committed to T4, use the `delete` command:
+You can save a Python object to a T4 file using the `put` command:
 
 ```python
-he.delete("bucket-name/my-frame.csv")
+import t4
+t4.put(df, "bucket-name/my-frame.parquet")
 ```
 
-### Versions
+The above code writes `df` at the top level of `bucket-name`.
+
+If you `put` to a folder that doesn't exist yet, `t4` will create that folder. If you overwrite an object, and bucket versioning is enabled, the overwritten object is retained as an older version of the same path. 
 
-In S3 buckets with object versioning enabled, object **versions** are automatic applied to every object in the bucket.
+T4 transparently serializes and de-serializes select Python
+objects. In the above example, `df` is automatically stored as an Apache
+Parquet file. This provides [substantial performance gains](http://wesmckinney.com/blog/python-parquet-update/).
```

Comment:
performance gains over exporting to CSV?

Reply:
I just omit this comment as it's irrelevant to the topic at hand anyway.